src/views/connectionUI.ts

```diff
@@ -73,10 +73,15 @@ export class ConnectionUI {
 
     // Helper to let user choose a connection from a picklist
     // Return the ConnectionInfo for the user's choice
-    public showConnections(): Promise<IConnectionCredentials> {
+    public showConnections(fromObjectExplorer: boolean = false): Promise<IConnectionCredentials> {
         const self = this;
```

Comment:
Why not just have the command from OE call self.createAndSaveProfile directly instead?

If you're really against that for some reason then I'd at least suggest making this name more generic - something like showExistingConnections = true